herring/herring/settings.py

```diff
@@ -148,6 +148,8 @@
 HERRING_SECRETS = json.loads(env.get_value('SECRETS', default='{}'))
 HERRING_FUCK_OAUTH = json.loads(env.get_value('FUCK_OAUTH', default='{}'))
 
+# This should be more dynamic in the future.
+HERRING_HUNT_ID = int(env.get_value('HUNT_ID', default=0))
```

Comment:
Any reason for this not to default to `datetime.now().year`? It would make rolling over the config variable one fewer thing to have to remember to do each year.